using-mobile-extensions/adobe-target/README.md

```diff
@@ -152,7 +152,7 @@ func application(_ application: UIApplication, didFinishLaunchingWithOptions lau
 
 {% tab title="React Native" %}
 
-To register the Target with Core, use the following API:
+To register the Target extension with the MobileCore Extension, use the following API:
```

Comment:
- please make "Extension" lower case --> "extension"
- Can we make "MobileCore" 2 word? --> "Mobile Core" (that's how i've typically seen it.